tess_area.cpp

```diff
@@ -40,6 +40,10 @@ int main(int argc, char** argv) {
 // only store the positive vertice id numbering
   
   while (inR.read_row(radi)) radius.emplace_back(radi);
+
+// find the maximum radius in order to ignore the left column and bottom layer when plotting postscript.
+  double max_radius = *(std::max_element(radius.begin(), radius.end()));
```

Comment:
const this

Reply:
Why are we ignoring the left and bottom layer?

Reply:
Because I saw your plotting(Fig 3.18) in your PhD thesis where the left and bottom layer are enclosed by voronoi tesselation but I ran my code and foundd there are no voronoi cells at these locations. So I have no idea where is wrong.

Reply:
We should include them